### ISSUE TITLE
Performance degradation

### ISSUE THREAD

**Issue by cwanja** [adriankumpf/teslamate#1632]
**Describe the bug**
Having issues with TeslaMate boggling down and performing slowly on the Pi. As soon as I enable it, the Pi comes to a crawl and is having issues just navigating the Pi GUI, terminals, windows, browser, etc. 

I did just re-install due to a failed SD card. Using a similar SD card and would expect the same performance. I previously set this up with version 2 of the YAML file - **would that make a difference?** I also imported my data from a backup.

I can also tell that if I stop TeslaMate entirely from docker, the performance snaps back to the way it should be. So I feel like I have contained it specifically to this installation. Just no idea why.

**Expected behavior**
Similar to before the SD card died, I would expect the Pi and TeslaMate to be performing well. I feel like the performance on the Pi is causing spotty data collection due to either write issues or something similar. SD card test on the Pi report it is performing well. 
```
Raspberry Pi Diagnostics - version 0.9
Tue May 18 14:58:25 2021

Test : SD Card Speed Test Run 1 
prepare-file;0;0;13047;25 
seq-write;0;0;17935;35 
rand-4k-write;0;0;2708;677 
rand-4k-read;7874;1968;0;0 
Sequential write speed 17935 KB/sec (target 10000) - PASS 
Random write speed 677 IOPS (target 500) - 
PASS Random read speed 1968 IOPS (target 1500) - PASS 

Test PASS
```

**How to reproduce it (as minimally and precisely as possible)**:
Probably impossible unless you want to hangout :) 

**Relevant entries from the logs**
Let me know if there are any desired logs. Not sure if it database, web application or what.

**Environment**

- TeslaMate version: 1.22.0
- Type of installation: Docker 
- OS TeslaMate is installed on: Raspbian OS
- User OS & Browser: iOS Safari, Windows 10 Chromium Edge
- Others: HALP!

**Comment by cwanja** [comment]
I want to say that this _might_ be resolved, but I am going to keep this open to get some opinions (if anyone has any). 

What I noticed was that the network was stuck on saying ```configuring```. I disabled the container, let the network configure and rebooted. It remained a ```configured``` post reboot and started the container. It seems that the performance sustained after starting up the container and about five minutes while monitoring it. I was able to type into the console without delay, move windows around, browse the folder hierarchy without lag and click with immediate mouse response.

I _want_ to say it is resolved. But I am still monitoring it. The trick with me if I go for a drive, what happens.
